media-libs/libjxl/libjxl-0.7.0_pre20220825.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib
+
+DESCRIPTION="JPEG XL image format reference implementation"
+HOMEPAGE="https://github.com/libjxl/libjxl"
+
+SRC_URI="https://api.github.com/repos/libjxl/libjxl/tarball/4c31ef03e3fd5239d6b794771d4ae7daa7815b28 -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="openexr"
```

Comment:
Make `LICENSE`, `SLOT`, `KEYWORDS`, `IUSE` their own block, in that order

Reply:
Thanks for the review. Please take a look again if I understood correctly.